README.md

```diff
@@ -307,6 +307,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [GitHub Action for Hexo](https://github.com/heowc/action-hexo)
 - [Deploy Google Analytics stats to GitHub Pages](https://github.com/cristianpb/analytics-google)
 - [A Jupyter Notebook Blogging Platform Powered by GitHub Actions, Pages and Jekyll](https://github.com/fastai/fastpages)
+- [A GitHub Action to deploy a static site on GitHub Pages (multiple repository)](https://github.com/appleboy/gh-pages-action)
```

Comment:
@appleboy  please change "A GitHub Action to deploy a static site on GitHub Pages (multiple repository)" to "Deploy A Static Site to GitHub Pages" and add a short description at the end "- Deploy to custom directory and ignore folder/file".

Reply:
@jessicalostinspace Done